src/app/components/styles/home-page-non-critical.scss

```diff
@@ -0,0 +1,6 @@
+.supporting-data-section {
+  h3 {
```

Comment:
Not sure if it's the right selector. When looking at the heading flow of the page here in this place we are using an h3 when what we should be using is a h2 so that will need to change at some point.

Reply:
Note that if you are going to use `tile__header` from Franklin, but switch this stylesheet to CSS module, you need to use this https://github.com/css-modules/css-modules#exceptions for global class names

Reply:
Also, should this actually be in franklin?

Reply:
This is not styling the tile header but making an exception for the supporting data container header. 
I will create a class called `.supporting-data-section__header` and assign put it on the `h3` as this will get the issue of tag specificity if we change the hierarchy one day

Reply:
Are you saying this is actually for https://www.ebi.ac.uk/panda/jira/browse/TRM-25709 ?

Reply:
No it's for this title 
![image](https://user-images.githubusercontent.com/3329765/112458055-49cef000-8d54-11eb-91ea-a7e4767b40df.png)

Reply:
![Screenshot 2021-03-25 at 10 27 58](https://user-images.githubusercontent.com/6025561/112458560-d2e62700-8d54-11eb-85c8-b4639a0d1bf9.png)

Reply:
Sorry, I need to get my glasses 🤓

Reply:
OK, so there it still needs to be an h2. If you need it to have the style of an h3 we should be theming it with Franklin mixins' `@include fs-font-settings('H3');`

Reply:
It's currently an h3 in all other sections - "News" "Analysis Tools" "Need Help etc...

I agree that the hierarchy of titles needs to be revisited across the website (the one that grinds my gears the most is the  entry page tab header being at the same level as the title in the overview) but this will be done after the alpha. I'd like to revisit the type sizing in Franklin too.

Reply:
I'm saying that because I'm changing the tile component at the moment, and including this in the batch of changes.